### PR TITLE
Fix Issue 1797

### LINKED ISSUE #1797: Membership Services CA env settings: "OBCCA_" -> "MEMBERSRVC_CA_"
## Description

The `membersrvc` process allows the configuration of environment variables that can override the YAML configuration settings.

Here's an excerpt from `docs/dev-setup/obcca-setup.md` ["perm"-linked](https://github.com/hyperledger/fabric/blob/f7b0b5c0a2fc98c898cd1972cc18c811d42c355e/docs/dev-setup/obcca-setup.md)

> Alternatively, these fields can be set via environment variables, which---if set---have precedence over entries in the yaml file. The correspnding environment variables are named as follows:
> 
> OBCCA_SERVER_GOMAXPROCS
> OBCCA_SERVER_ROOTPATH
> OBCCA_SERVER_CADIR
> OBCCA_SERVER_PORT
## Proposed Change

Currently we have other libraries using `HYPERLEDGER` `CORE_NOOPS`, `CORE_PBFT` as a prefix. We could possibly use `HLF_CA_` as a prefix, which is also in line with the entire Certificate Authority (CA) functionality of `Membership Services`.

> HLF_CA_SERVER_GOMAXPROCS
> HLF_CA_SERVER_ROOTPATH
> HLF_CA_SERVER_CADIR
> HLF_CA_SERVER_PORT
## Good First Bug

I consider this a **Good First Bug** especially for those in the community who would like to pick up a simple coding task and get involved in fabric, build it, test it, and learn how to contribute to it. Need this done quickly and will be happy to spend some time guiding (pointing to the right Wiki's mostly ;-)), and verifying this all the way to submitting the PR [ usually at the modest tuition charge of 1-2 `MoreTests` digital currency ;-) ]

Interested? Hit me/us up here, or on [HL's Slack](http://hyperledgerproject.slack.com/jonathanlevi)
## How to Fix

1. The good ol' `membersrvc/ca/util...` should use the new updated prefix (and not the `OBC...`)
2. Use `viper`'s `config.SetEnvPrefix(configPrefix)` (in one location) instead of manual string concatenations.
3. Rename `dev-setup/obcca-setup.md` to `dev-setup/ca-setup.md` and related references
   and of course,
4. Update  `dev-setup/ca-setup.md` content

### PR DESCRIPTION
**Pull request by GrapeBaBa**
## Description

Fix issue 1797 on demand.
## Motivation and Context

Fixes #1797 
## How Has This Been Tested?

unit test and hehave test done.
## Checklist:

- [x] I have added a [Signed-off-by](https://github.com/hyperledger/fabric/blob/master/CONTRIBUTING.md#legal-stuff)
- [x] Either no new documentation is required by this change, OR I added new documentation
- [x] Either no new tests are required by this change, OR I added new tests
- [x] I have run [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports), [go vet](https://golang.org/cmd/vet/), and [golint](https://github.com/golang/lint). I have cleaned up all valid errors and warnings in code I have added or modified. These tools may generate false positives. Don't be worried about ignoring some errors or warnings. The goal is clean, consistent, and readable code.

Signed-off-by:Kai Chen ckaiwh@cn.ibm.com

### CODE REVIEW

**Comment by GrapeBaBa** [comment]
How to re-trigger the travis build?

**Comment by JonathanLevi** [comment]
reverify Travis

**Comment by JonathanLevi** [comment]
So 'reverify Travis' doesn't work, though 'reverify jenkins'' used to.

The simplest (old fashioned, but working) is to push something to the same branch... (Can be more than one commit at a time, in case you want to do and undo a change, or so)

**Comment by GrapeBaBa** [comment]
@JonathanLevi For this PR currently I have no more code need to push, unit test passed in my local, I just want to trigger the build again.

**Comment by JonathanLevi** [comment]
BTW: I am not in front of a PC/Mac and w/o my glasses ;-), but given #1803 (and #1809 ), the build is probably not broken due to your current changes. I believe this will be resolved tomorrow, which will affect/fix several pending PRs....

Let's give it a day?

**Comment by GrapeBaBa** [comment]
Sure, enjoy your weekend.

File: docs/dev-setup/ca-setup.md [hunk 1]
```diff
@@ -35,10 +35,10 @@ Furthermore, logging levels can be enabled/disabled by adjusting the following s
 
 Alternatively, these fields can be set via environment variables, which---if set---have precedence over entries in the yaml file.  The correspnding environment variables are named as follows:
```
Comment (kchristidis) [inline docs/dev-setup/ca-setup.md]: s/correspnding/corresponding

**Comment by bramwelt** [comment]
reverify jenkins

**Comment by bramwelt** [comment]
reverify jenkins

**Comment by JonathanLevi** [comment]
Good to know @christo4ferris, thanks.

@mastersingh24: can we have your 'blessing' on this? (we initially talked about "CA_..." then on #fabric-dev it was suggested to use "HLF_CA_"... so that the "CORE_" will later change to "HLF_CORE_").

Either way, it's good to update the "OBC" prefix for starters, and remove the 2 not-anymore-necessary `ca/util.go` helper functions...

**Comment by JonathanLevi** [comment]
Good to know @christo4ferris, thanks. Suggested to add these to the Wiki in #1817/#1816.

@mastersingh24: can we have your 'blessing' on this? (we initially talked about "CA_..." then on #fabric-dev it was suggested to use "HLF_CA_"... so that the "CORE_" will later change to "HLF_CORE_").

Either way, it's good to update the "OBC" prefix for starters, and remove the 2 not-anymore-necessary `ca/util.go` helper functions...

**Comment by JonathanLevi** [comment]
Looks good to me (see also notes on the #1810). @christo4ferris, @srderson @ghaskins: mind merging this too (regardless of the known #680 failure)?

Not a biggy, but helps getting rid of more redundant functions (in `membersrvc/ca/util.go`) and the old OBC... refs.

**Comment by srderson** [comment]
are we waiting for comment from @mastersingh24?

**Comment by JonathanLevi** [comment]
@srderson Not really/anymore.

We had a discussion on #fabric-dev, and I discussed it with @mastersingh24 (after asking for his blessings above), considering everything, @mastersingh24/Gari suggested that we are good to go with the  `MEMBERSRVC_CA_` prefix instead of the original `HLF_CA_` (to which @ghaskins agreed as well).

**Comment by mastersingh24** [comment]
I am good

**Comment by srderson** [comment]
LGTM. Thanks @GrapeBaBa

**Pull request merged**